src/tests/flask/routes/chef_api_e2e_test.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy import text
+
+from src.tests.flask.base_flask_setup_test import BaseFlaskSetupTest
+from src.tests.utils.fixtures.json_entities_fixture import build_chef
+
+QUERY_TO_INSERT_AN_ORDER = """
+  CALL insert_order_with_defaults();
+"""
+
+
+class ChefApiE2ETest(BaseFlaskSetupTest):
+    def after_base_setup(self):
+        self.json_chef = build_chef()
+
+    def test_add_chef_request(self):
+        request = self.client.post("/chefs", json=self.json_chef)
+        self.assertEqual(request.get_json(), self.json_chef)
+
+    def test_get_by_id_request_successfully(self):
+        chef_1 = self.client.post("/chefs", json=self.json_chef)
+        request = self.client.get("/chefs/1")
+        self.assertEqual(request.get_json(), chef_1.get_json())
+
+    def test_get_all_chef_request_successfully(self):
+        chef_1 = self.client.post("/chefs", json=self.json_chef)
+        chef_2 = self.client.post("/chefs", json=self.json_chef)
+        chef_3 = self.client.post("/chefs", json=self.json_chef)
+
+        request = self.client.get("/chefs")
+
+        self.assertEqual(
+            request.get_json(),
+            [chef_1.get_json(), chef_2.get_json(), chef_3.get_json()],
+        )
+
+    def test_updated_chef_request_successfully(self):
+        self.client.post("/chefs", json=self.json_chef)
+        chef_with_parameter_updated = {"skill": 10, "updated_by": 2}
+        request = self.client.put("/chefs/1", json=chef_with_parameter_updated)
+        request_updated_parameter = request.get_json()
+        self.assertEqual(10, request_updated_parameter["skill"])
+
+    def test_deleted_chef_request_successfully(self):
+        self.client.post("/chefs", json=self.json_chef)
+        request = self.client.delete("/chefs/1", json={"updated_by": 3})
+        self.assertEqual(request.get_data(as_text=True), "Chef 1 Deleted")
+
+    def test_get_available_chef_request_successfully(self):
+        with self.engine.begin() as conn:
```

Comment:
Move this to the order fixture, and please pass down arguments to set the different props, in this case status

Reply:
done.